kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wlst/create-domain.py

```diff
@@ -25,6 +25,7 @@ def getEnvVar(var):
 managed_server_name_base     = getEnvVar("MANAGED_SERVER_NAME_BASE")
 managed_server_name_base_svc = getEnvVar("MANAGED_SERVER_NAME_BASE_SVC")
 domain_logs                  = getEnvVar("DOMAIN_LOGS_DIR")
+use_default_logs_dir         = getEnvVar("USE_DEFAULT_LOGS_DIR")
```

Comment:
There's no need for create-domain.py to set Log locations since the introspector will override via sit-config.